docs/docs-content/automation/palette-cli/commands/vmo.md

```diff
@@ -117,6 +128,125 @@ palette vmo import-ova --skip-image
 
 ## Migrate VM
 
+### Prerequisites
```

Comment:
These are so long! Should we maybe just link over to the full guide instead?

Reply:
That may be the better option  and just state something like  "Refer to ...... for a comprehensive list of requirements. Something along those lines".

Reply:
Have added now!